swagger.yaml

```diff
@@ -207,12 +206,13 @@ definitions:
   NewImage:
     type: object
     description: "New Image metadata model, used to create new images. The image ID will be created by image API, and the state will be set to 'created' at creation time."
+    required:
```

Comment:
Isn't the collection ID optional? It is marked here as required. (is that something about how the swagger spec works?)

Reply:
The query parameter is optional (it used to be compulsory but we needed to change it).
But the `collectionId` field in the body is compulsory when we post a new image - maybe I can ask James if we want to make it optional too.